### ISSUE TITLE
optimization_problem.initial_state() missing alias relation

### ISSUE THREAD

**Issue by SGeeversAtVortech** [Deltares/rtc-tools#1030]
In GitLab by @jvande42b on May 16, 2018, 16:50

Currently:

``` python
return AliasDict(
    {variable: self.interpolate(t0, timeseries.times, timeseries.values)
     for variable, timeseries in history.items()})
```

Probably should be (?):

``` python
return AliasDict(
    self.alias_relation,
    {variable: self.interpolate(t0, timeseries.times, timeseries.values)
     for variable, timeseries in history.items()})
```

If not, at least needs comment explaining why.

https://gitlab.com/deltares/rtc-tools/blob/master/src/rtctools/optimization/optimization_problem.py#L406

**Comment by SGeeversAtVortech** [comment]
In GitLab by @jvande42b on May 17, 2018, 17:24

mentioned in commit 49d948f0da848fd09af2d6f0e99b6dc7c36f62ae

**Comment by SGeeversAtVortech** [comment]
In GitLab by @jvande42b on May 17, 2018, 17:41

mentioned in commit 12fb087ac5b5a400a4c6562139162eefb0f7409e

**Comment by SGeeversAtVortech** [comment]
In GitLab by @jvande42b on May 17, 2018, 17:44

mentioned in commit 965db9d67d0694b180e6bb74e112044205150cc4

**Comment by SGeeversAtVortech** [comment]
In GitLab by @jvande42b on May 17, 2018, 17:47

mentioned in commit 12902ac58fcdab17d322563bca90278fd81ae48e

**Comment by SGeeversAtVortech** [comment]
In GitLab by @jvande42b on May 17, 2018, 18:45

closed via commit 12902ac58fcdab17d322563bca90278fd81ae48e

**Comment by SGeeversAtVortech** [comment]
In GitLab by @jvande42b on May 17, 2018, 18:45

closed via merge request !118
